protocol/cmd/dydxprotocold/cmd/root.go

```diff
@@ -165,6 +165,7 @@ func NewRootCmdWithInterceptors(
 	if err != nil {
 		panic(err)
 	}
+	initClientCtx.HomeDir = ""
```

Comment:
is there a reason why it is set here instead of above on line 109?

If so can you document why with a comment?

Reply:
If we set this before `ReadFromClientConfig`, the config will be created in cwd instead.